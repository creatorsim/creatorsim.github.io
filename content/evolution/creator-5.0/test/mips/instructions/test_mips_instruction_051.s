
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
    main:
      li  $t0, 1
      li  $t1, 0
      li  $t2, -1
      beqz $t0, jump1
      beqz $t2, jump1
      beqz $t1, jump1

    jump2:
      li $t6, 34
      li $v0, 10
      syscall

    jump1:
      li $t4, 0
      beqz $t4, jump2
