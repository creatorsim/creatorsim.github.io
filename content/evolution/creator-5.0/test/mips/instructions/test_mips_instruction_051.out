
cr[PC]:0x2c; ir[1,at]:0xffffffff; ir[2,v0]:0xa; ir[8,t0]:0x1; ir[10,t2]:0xffffffff; ir[14,t6]:0x22; keyboard[0x0]:''; display[0x0]:''; 

